.github/workflows/build.yml

```diff
@@ -340,3 +369,44 @@ jobs:
       - name: copy binaries to s3
         run: |
           aws s3 sync . s3://apex-net/
+
+  build-rpm:
+    needs: ["go-lint"]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: docker/setup-buildx-action@v2
```

Comment:
Is buildx used in this job?

Reply:
docker build is used, at least. I copied it from other jobs that were doing some image builds. What I really want is "please set up docker so I can `docker build` and `docker run` in later steps."

Reply:
You don't need that step then since both of those are available on the runner itself.

Reply:
fixed